docs/contributors.yaml

```diff
@@ -402,13 +402,13 @@
   jira_user: yanyanhu
   reviewboard_user: yanyanhu
 
-- name: Yong Qiao Wang
+- name: Yongqiao Wang
   affiliations:
-    - {organization: IBM}
```

Comment:
Instead of removing your old affiliation (IBM) completely, can you add an "end_date" to it? That way your contributions while you were employed at IBM, will be attributed correctly. See my affiliation above for an example.